src/server/gc_worker/rawkv_compaction_filter.rs

```diff
@@ -136,13 +180,71 @@ impl CompactionFilter for RawCompactionFilter {
     }
 }
 
+fn do_check_allowed(enable: bool, skip_vcheck: bool, feature_gate: &FeatureGate) -> bool {
+    enable && (skip_vcheck || feature_gate.can_enable(COMPACTION_FILTER_RAWKV_GC_FEATURE))
+}
+
+fn check_need_gc(
```

Comment:
Suggest to reuse `check_need_gc` of TiDB's compaction filter, which can adapt RawKV's MvccProperties as `props.min_ttl_ts` is always 0.

Reply:
Now, It reuse check_need_gc.

And ```props.min_ttl_ts``` default value is ```TimeStamp::max()```.
So,if props.min_ttl_ts != TimeStamp::max() && props.min_ttl_ts > safe_point, then it will return (false,false)
If ```props.min_ttl_ts = TimeStamp::max()``` mean there are no writes with ttl, the return value depends on other conditions.